src/main/java/rife/cmf/loader/xhtml/SAXLoader.java

```diff
@@ -134,7 +131,7 @@ public LoadedContent<String> load(String data, boolean fragment, Set<String> err
 
             if (sax_parse_exception ||
                 (errors != null &&
-                 errors.size() > 0) ||
+                    !errors.isEmpty()) ||
```

Comment:
Indentation is wrong

Reply:
Actually that one is right, it's indented based on the fact that is enclosed in (). The indentation of the one two lines below is wrong.